File: Chapter08_Performance/speed2.c
#include <stdio.h>
#include <math.h>

int main() {
  int i,j,k=0;
  for(i=0;i<100000;i++) {
    for(j=1;j<10000;j++) {
      k=i/j;
    }
  }
  printf("%d\n",k);
  return 0;
}

File: Chapter08_Performance/speed2.py
import math
import numpy

a = 1.
for i in numpy.arange(1,1000000000/100):
    #a=a+12.1
    #a=a*1.0000001
    #a=a/1.0000001
    a=math.sin(3.*a)
    #a=numpy.sin(3.*a)
    
print a
